azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -1005,6 +1005,7 @@ private void ajaxExecuteFlow(final HttpServletRequest req,
     }
 
     final ExecutableFlow exflow = FlowUtils.createExecutableFlow(project, flow);
+    exflow.setUploadUser(project.getUploadUser());
```

Comment:
Could you please also add this line in ReportalServlet.java?
(Before .setExecutionSource() line)

Reply:
fixed. good catch !